snap/snapcraft.yaml

```diff
@@ -0,0 +1,35 @@
+name: lazygit
+title: Lazygit
+base: core24
+adopt-info: lazygit
+summary: simple terminal UI for git commands
+description: A simple terminal UI for git commands, written in Go with the gocui library.
+version: '0.42.0'
+license: MIT
+grade: stable
+confinement: classic
+compression: lzo
+contact: 
+  - https://discord.gg/ehwFt2t4wt
+  - https://twitter.com/DuffieldJesse
+donation: 
+  - https://github.com/jesseduffield
+  - https://donorbox.org/lazygit
+issues: https://github.com/jesseduffield/lazygit/issues
+source-code: https://github.com/jesseduffield/lazygit
+website: https://jesseduffield.com
+
+parts:
+  lazygit:
+    plugin: go
+    source: https://github.com/jesseduffield/lazygit.git
+    source-tag: 'v$SNAPCRAFT_PROJECT_VERSION'
+    source-depth: 1
+    build-snaps:
+      - go
+    build-attributes:
+      - enable-patchelf
```

Comment:
Can you please explain what this patchelf thing does.

Reply:
Because this snap will be a classic snap, there will no confinement for this snap working. So, this snap will be mounted in the `usr` namespace, and thus the rpaths might point to libraries from your system. patchelf is used to patch those elf-interpretors to point towards proper rpaths of the libraries shipped by the snap.